addons/wowl/static/tests/webclient/navbar_tests.ts

```diff
@@ -36,7 +37,11 @@ QUnit.module("Navbar", {
       Component: MySystrayItem,
     };
     systray.add(item.name, item);
-    baseConfig = { services, serverData, systray };
+    const browser: Partial<OdooBrowser> = {
+      setTimeout: (handler, delay, args) => (handler as Function).bind(this, args)(),
```

Comment:
Doesn't this work?
`setTimeout: (handler: Functon, delay, args) => handler(args),`